### ISSUE TITLE
list of drive managed keys command

### ISSUE THREAD

**Issue by immetoo2** [holepunchto/drives#19]
After touching and/or downloaded a multiple drives, it manages those keys but is missing a listing command, for example;
```
drives blkid
```
Results in one key per line output.

**Comment by LuKks** [comment]
Yep, it's actually a Corestore limitation that doesn't have a method to list the managed Hypercores. Will most definitively be possible in the future

**Comment by arunkd13** [comment]
Since there is no concrete date available for https://github.com/holepunchto/corestore/issues/6 I think it is simpler to track the list of drives created, outside of the corestore.

https://github.com/holepunchto/drives/pull/32 uses the above approach.

**Comment by immetoo2** [comment]
Since i tried to use ```drives``` in build pipeline steps and unit tests, my amount of junk drives grows per failed unit test.
A managed way to delete those manually without deleting everything, would be very appreciated.
